test/configCases/externals/externals-scoped/index.js

```diff
@@ -0,0 +1,5 @@
+it("should not fail compilation code with scoped external", function(done) {
+	// eslint-disable-next-line node/no-missing-require
+	require("@scoped/package");
```

Comment:
this test case is still not working, but it fails at runtime now - instead during parsing